src/Symfony/Component/Serializer/Tests/Encoder/YamlEncoderTest.php

```diff
@@ -54,14 +54,20 @@ public function testSupportsDecoding()
         $this->assertFalse($encoder->supportsDecoding('json'));
     }
 
+    /**
+     * The legacy group must be reverted on 3.4 and the php/object tag without a following colon must be used instead.
+     * @group legacy
```

Comment:
This is not a nice solution. But unfortunately we cannot detect whether we use version 3.3 or 3.4 of the Yaml component.